README.md

```diff
@@ -18,6 +29,58 @@ Or install it yourself as:
 
 ## Usage
 
+### Configuration in Rails
+
+The goal: constrain all configuration regarding the logging to go through
+logatron and not the internals of logatron (such as lograge). Consumers of
+logatron should not have to know the internals.
+
+Logatron must be configured before anything else since there can be other
+configuration dependent upon those values (lograge for sure). This is done by
+doing that configuration in an initializer file that will be listed first when
+sorted in that directory alphanumerically. Putting an underscore (`_`) in front
+of the name typically will move it to the top of the list - 
+`config/initializers/_log.rb`. Here is an example configuration.
+
+```ruby
+require 'logatron/logatron'
+
+Logatron.configure do |config|
+  log_name = 'quartermaster'
+  if ENV['DAEMON']
+    script  = File.basename(ENV['DAEMON'])
+    script.slice! File.extname(script)
+    log_name = script
+  end
+  destination = Rails.env == 'test' ? '/dev/null' : STDOUT
+  config.host = `hostname`.chomp
+  config.app_id = log_name
+  config.logger = Logger.new(destination)
+  config.level = Logatron::INFO
+  config.base_controller_class = 'ActionController::API'
+  config.add_rails_request_field(:user_agent, &:user_agent)  # optional
+end
+```
+
+The `ENV['DAEMON']` logic is for Ascent when deploying as an AMQP daemon
+instead of a rails app (outside the scope of this README.md).
+
+All the configuration fields above are required with the exception of
+`.add_rails_request_field`. That method is an extension point for the
+auto-logging of rails HTTP requests. If there is an additional parameter needed
+in the log line of the rails HTTP request, that can be specified here. The
+first argument is the name you want that new field to have in the rails HTTP
+request log line. The second argument is a block that takes 
+`ActionDispatch::Request` as an optional parameter and returns the value to put
+next to the display name in the rails HTTP request log line. The
+`.add_rails_request_field` method can be called 0 or more times, once per
+additional field to put in the log line.
+
+As of now, any additional fields added to the rails HTTP request log line
+affect all routes in all controllers. There is no way currently to log extra
+fields on specific controller actions. In other words, the configuration is 
+global to all routes.
```

Comment:
`config.base_controller_class` is new, right? should probably add a short paragraph about why it needs to be specified, and perhaps why it might be something other than `ActionController::API`.